airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -111,12 +112,19 @@ const ConnectorList: React.FC<MenuWithRequestButtonProps> = ({ children, ...prop
   </>
 );
 
-const StageLabel: React.FC<{ releaseStage?: ReleaseStage }> = ({ releaseStage }) =>
-  releaseStage && releaseStage !== ReleaseStage.GENERALLY_AVAILABLE ? (
+const StageLabel: React.FC<{ releaseStage?: ReleaseStage }> = ({ releaseStage }) => {
+  if (!releaseStage) return null;
```

Comment:
Nitpick: Looks like we wrap one-line if's in `{}`
```suggestion
  if (!releaseStage) {
    return null;
  }
```